src/graphql/buy-entry.ts

```diff
@@ -92,4 +97,17 @@ const buyEntry = {
   },
 };
 
+const deleteEntryCache = (code: string, issuer: string) => {
+  const assetId = `${code}-${issuer}`;
+  const baseURL = 'https://api.stellar.expert/explorer/';
+
+  const urls = [
+    `${baseURL}${Config.STELLAR_NETWORK}/asset/${assetId}/history/all?limit=100`,
+    `${baseURL}${Config.STELLAR_NETWORK}/asset/${assetId}/holders?limit=100`,
+    `${baseURL}${Config.STELLAR_NETWORK}/asset/${assetId}/history/offers?limit=100&order=desc`
+  ];
+
+  deleteCache(urls);
```

Comment:
Note: this function works for given urls, because axios-cache-adapter uses url + query params string as keys when request has empty body. For request with non empty body it adds MD5 hash of body.